content/en/docs/instrumentation/net/automatic/config.md

```diff
@@ -0,0 +1,352 @@
+
+The following resource detectors are included and enabled by default:
+
+| ID                | Description                | Documentation                                                                                                                                                                                                     | Status                                                    |
+| ----------------- | -------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------- |
+| `CONTAINER`       | Container detector         | [Container resource detector documentation](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/ResourceDetectors.Container-1.0.0-beta.3/src/OpenTelemetry.ResourceDetectors.Container/README.md) | [Experimental](/docs/specs/otel/versioning-and-stability) |
+| `AZUREAPPSERVICE` | Azure App Service detector | [Azure resource detector documentation](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/ResourceDetectors.Azure-1.0.0-beta.2/src/OpenTelemetry.ResourceDetectors.Azure/README.md)             | [Experimental](/docs/specs/otel/versioning-and-stability) |
+
+## Propagators
+
+Propagators allow applications to share context. See
+[the OpenTelemetry specification](/docs/specs/otel/context/api-propagators) for
+more details.
+
+| Environment variable | Description                                                                                                                                                                                                                                                                                                  | Default value          |
+| -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------- |
+| `OTEL_PROPAGATORS`   | Comma-separated list of propagators. Supported options: `tracecontext`, `baggage`, `b3multi`, `b3`. See [the OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.14.0/specification/sdk-environment-variables.md#general-sdk-configuration) for more details. | `tracecontext,baggage` |
+
+## Samplers
+
+Samplers let you control potential noise and overhead introduced by
+OpenTelemetry instrumentation by selecting which traces you want to collect and
+export. See
+[the OpenTelemetry specification](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.15.0/specification/sdk-environment-variables.md?plain=1#L45-L80)
```

Comment:
Replace this with a link to the spec within the website, this also points to an fairly old version of the spec.
```suggestion
[the OpenTelemetry specification](/docs/specs/otel/configuration/sdk-environment-variables/#general-sdk-configuration)
```